scripts/modules/validation/sanitizers.js

```
@@ -0,0 +1,667 @@
+			modelConfig.provider = modelConfig.provider.toLowerCase().trim();
+		}
+
+		// Ensure numeric values are actually numbers
+		if (modelConfig.maxTokens) {
+			modelConfig.maxTokens = Number(modelConfig.maxTokens);
+			if (
+				!Number.isInteger(modelConfig.maxTokens) ||
+				modelConfig.maxTokens < 1
+			) {
+				log('warn', `Invalid maxTokens for ${role}, using default`);
+				delete modelConfig.maxTokens;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)